sdk/core/core-amqp/src/util/utils.ts

```diff
@@ -58,7 +58,7 @@ export interface WebSocketOptions {
  * A constant that indicates whether the environment is node.js or browser based.
  */
 export const isNode =
-  !!process && !!process.version && !!process.versions && !!process.versions.node;
+  typeof process !== "undefined" && Boolean(process.version) && Boolean(process.versions?.node);
```

Comment:
This change is needed to make the encoder browser tests that import event-hubs work. We did not see this issue in event hubs browser tests because they're skipped for some reason. This definition is copied from `@azure/core-util's isNode`.